specs/dxgi.py

```diff
@@ -1149,7 +1150,13 @@
     StdMethod(HRESULT, 'EnumAdapterByGpuPreference', [(UINT, 'Adapter'), (DXGI_GPU_PREFERENCE, 'GpuPreference'), (REFIID, 'riid'), Out(Pointer(ObjPointer(Void)), 'ppvAdapter')]),
 ]
 
+IDXGIFactory7.methods += [
+    StdMethod(HRESULT, 'RegisterAdaptersChangedEvent', [(HANDLE, 'hEvent'), (Pointer(DWORD), 'pdwCookie')], sideeffects=False),
```

Comment:
Should be `Out(Pointer(DWORD), 'pdwCookie')`, to tell it's an output parameter.

Reply:
I went on a vacation shortly after this and forgot, oops! Will fix tomorrow.

Reply:
Fixed!